js/ccxt.js

```diff
@@ -0,0 +1,386 @@
+/*
+
+MIT License
+
+Copyright (c) 2017 Igor Kroitor
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in all
+copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+SOFTWARE.
+
+*/
+//-----------------------------------------------------------------------------
+/* eslint-disable */
+import { Exchange } from './src/base/Exchange.js';
+import { Precise } from './src/base/Precise.js';
+import * as functions from './src/base/functions.js';
+import * as errors from './src/base/errors.js';
+//-----------------------------------------------------------------------------
+// this is updated by vss.js when building
+const version = '2.5.97';
+Exchange.ccxtVersion = version;
```

Comment:
I think here we would break this snippet that used to work in the examples:

```JavaScript
import ccxt from 'ccxt'
console.log ('CCXT Version:', ccxt.version)
```

Lmk if I'm missing something.

Reply:
@kroitor the thing I did not change that, it's what we currently have in master: https://github.com/ccxt/ccxt/blob/master/ccxt.js#L42
But sure I will double check it to see if it works or not

Reply:
@kroitor I  checked it and it works just fine because we do this at the bottom of the file 
```Javascript
const ccxt = Object.assign({ version, Exchange, Precise, 'exchanges': Object.keys(exchanges), 'pro': pro }, exchanges, functions, errors);
```
⬆️  assign some variables like `version` to the default exported object `ccxt`